shared/persistence-model/src/main/java/gov/va/vro/persistence/model/ContentionEventEntity.java

```diff
@@ -0,0 +1,27 @@
+package gov.va.vro.persistence.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "bie_contention_event")
+public class ContentionEventEntity extends BaseEntity {
+
+  @Column(name = "notified_at")
+  private LocalDateTime notifiedAt;
+
+  @Column(name = "event")
+  private String eventType;
+
+  @Column(name = "event_details")
+  private String eventDetails;
```

Comment:
How will we prevent PII from getting stored as `event_details`?

Reply:
I will update this PR with hardcoded filler for now with a comment to update.